src/credits/credits.c

```diff
@@ -825,7 +825,8 @@ static void ov76_021E68C8(CreditsAppWork *work) {
     ptr->count++;
 }
 
-static void FlipScreensCB(int a0, ScreenFlipWork *a1, int a2) {
+static void FlipScreensCB(SysTask *a0, void *a1v) {
+    ScreenFlipWork *a1 = (ScreenFlipWork *)a1v;
```

Comment:
document a1, phase out work terminology

Reply:
Phasing out work terminology is out of scope